TodoApp/TodoApp.Api/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Description;
+using Microsoft.Web.Http;
+using TodoApp.Api.Models;
+
+namespace TodoApp.Api.Controllers
+{
+    [ApiVersion("1.0")]
+    [RoutePrefix("api/v{version:apiVersion}/items")]
+    public class ItemsController : ApiController
+    {
+        internal static readonly Item[] Items = 
```

Comment:
Make this private and don't make it visible for tests. Make your tests more dummy and only check what is necessary.